pkgs/applications/virtualization/docker/default.nix

```diff
@@ -11,14 +11,14 @@ with lib;
 
 stdenv.mkDerivation rec {
   name = "docker-${version}";
-  version = "1.13.1";
-  rev = "092cba3"; # should match the version commit
+  version = "17.03.0-ce";
+  rev = "60ccb22"; # should match the version commit
```

Comment:
@copumpkin it's required to for the ` Git commit:   092cba3` part of the `docker version` output (and I think build might fail if `DOCKER_GITCOMMIT` is not there)